client/src/components/profile/Comment.jsx

```diff
@@ -1,52 +1,52 @@
 import React, { useEffect, useState } from 'react';
-import axios from 'axios';
 import './Profile.css';
 import { Button } from 'react-bootstrap'
+import { useSelector } from "react-redux"
 
-const Comment = () => {
-    const [comment, setComment] = useState('');
+const Comment = ({ post, setCommentFlag }) => {
+  const [comment, setComment] = useState('');
 
-// handle change fn
-// handle submit fn
-const submitComment = () => {
-  // post on b/e routes
- axios.post('http://localhost:5000/comments', 
- {comment_content:comment //dbname: clname
-}).then(() => {
-  alert('successful insert');
-})
+  const userId = useSelector(state => state.user.user_id)
 
-};
+  const [comments, setComments] = useState([])
 
-const commentData = async () => {
-        const res = await axios.post("https://jsonplaceholder.typicode.com/posts", {comment})
-        .then( res => {
-          console.log(res.data);
-          console.log(res);
-        });
-};
+  const submitComment = () => {
+    // post on b/e routes
+
+    setCommentFlag(state => !state)
 
-useEffect(() => {
-    commentData();
-}, []);
+    fetch('/api/post/comment', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        comment, postId: post.post_id, userId
+      }),
+    })
+      .then(response => response.json())
+      .then(data => {
+        console.log(data);
+      });
+  }
 
-return (
-    <div>
+  return (
+    <div> 
 
       <div className="card-comment">
-      
-      <form className='form-comment'>
-        <label style={{color:'#FCA73D'}}>
-         Leave a Reply?
-        </label><br/>
-        <input type="text" name="comment" onChange={(e) => { 
-setComment(e.target.value)
-          }} /> 
-        <Button style={{margin:'auto', marginTop:'10px', color:'white'}} onClick={submitComment}>Post A Comment</Button>
-      </form>
-        
+
+        <form className='form-comment'>
+          <label style={{ color: '#FCA73D' }}>
+            Leave a Reply?
+        </label><br />
+          <input type="text" name="comment" onChange={(e) => {
+            setComment(e.target.value)
```

Comment:
User will leave multi-line text input field  better way to use  <textarea> element instead of input element. When user want to leave a long comment he/she cant read their comment while typing.

Reply:
I changed with textarea but when I click enter in `textarea` it goes to the bottom line, I'll find a solution for it